project_github/project_github.api.php

```diff
@@ -70,3 +70,25 @@ function hook_github_project_validate_release(Node $release_node, array &$errors
     $errors['project_type_invalid'] = t('Project releases are only supported on module projects.');
   }
 }
+
+/**
+ * Modify the contents of the GitHub webhook payload.
+ *
+ * @param $project_name
+ *   The name of the project.
+ * @param array $files
+ *   An array representing the files in the payload directory after the payload
+ *   has been unzipped, and the original zip file deleted, and before the new
+ *   archive is created and resent to GitHub.
+ * @return NULL
+ *   No return value.
+ */
+function hook_project_github_create_package_alter($project_name, array $files) {
```

Comment:
```suggestion
function hook_project_github_create_package_alter($project_name, array &$files) {
```

I think $files has to be passed by reference in order for this to work.

Reply:
Hm, and actually most alter hooks have the array that is altered as the first parameter. e.g. hook_form_alter() passes $form, $form_state, and $form_id. This is similar where the $form_id is passed last even though it is the identifier.

Reply:
Fixed in new commit.